src/wavm/openmp.cpp

```diff
@@ -66,7 +72,7 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
                                omp_get_thread_num)
 {
     faabric::util::getLogger()->debug("S - omp_get_thread_num");
-    return thisThreadNumber;
```

Comment:
The OpenMP context was previously held in a number of global thread-local variables, so I've now encapsulated this into an `OpenMPContext` class accessible via the `getOpenMPContext()` method